src/components/Feed/Socialize/AdmireLine.tsx

```diff
@@ -0,0 +1,91 @@
+import { useFragment } from 'react-relay';
```

Comment:
this file is for this component 
<img width="848" alt="image" src="https://user-images.githubusercontent.com/6754223/195633246-1f11cebe-5202-49e6-b1f1-cc942b86be70.png">